.github/workflows/tests.yml

```diff
@@ -0,0 +1,79 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: continuous-integration
+
+on:
+  push:
+    branches: [master]
+    tags:
+      - 'v*'
+  pull_request:
```

Comment:
```suggestion
  pull_request:
    branches: [master]
```

Reply:
any specific reason to restrict to master?

Reply:
No particular reason. I just thought it would only run if you specified a target branch.

Reply:
no it runs on all branches if you don't specify 😄